GradeCenter.Services/Schools/ISchoolService.cs

```diff
@@ -0,0 +1,18 @@
+﻿using GradeCenter.Data.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GradeCenter.Services.Schools
+{
+    public interface ISchoolService
+    {
+        School? GetSchoolById(int id);
+        Task Create(string newName, string newAddress, int principalId, string principalFirstName, string principalLastName);
+        Task Update(int id, string newName, string newAddress);
```

Comment:
Change this to Task Create(SchoolUpdateRequestModel requestModel)

Reply:
Same comment a

> Currently, including the request model in the GradeCenter.Services project would not be possible, as the class file resides within the GradeCenter.API project.
> 
> Including a project reference from the GradeCenter.Services to the GradeCenter.API project will cause circular dependency.

Same comment as my reply regarding the Create method. Just change the signature to accept "School"  instead. :)